examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -46,7 +39,12 @@ export interface ITaskEvents extends IEvent {
     /**
      * Emitted when the name or priority have changed respectively.
      */
-    (event: "nameChanged" | "priorityChanged", listener: () => void);
+    (event:
```

Comment:
I know you said this is a hack, but nit here, probably preferrable to have each listener be declared separately even if the signature for all of them is the same. I personally find it cleaner, and I think there was some advantage for automated documentation... @Josmithr I think it was you pointing that out in other PRs I've seen in the past?

Reply:
How do I declare each listener separately? As far as I can tell, this is the pattern we've been using in this repo, so I've just stuck to is. Is there an example you can link me to that declares each listener separately? :)

Reply:
Just split each event into its own full line instead of using the union operator for the values of `event`, you can see an example [here](https://github.com/microsoft/FluidFramework/blob/main/packages/common/container-definitions/src/loader.ts#L151-L174).

Reply:
Oh, and the example uses `/* eslint-disable @typescript-eslint/unified-signatures */` and a corresponding `eslint-enable` after that interface declaration because eslint prefers them unified. And I found where Josh [suggested](https://teams.microsoft.com/l/message/19:50292e8934024fc19d6ca2080dd7681e@thread.skype/1666116316603?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=9ce27575-2f82-4689-abdb-bcff07e8063b&parentMessageId=1666116316603&teamName=Fluid%20Framework&channelName=Dev&createdTime=1666116316603&allowXTenantAccess=false) we make each listener its own thing, mostly for documentation purposes, which I agree with.

Reply:
Got it! Updated! Thanks for all the links and explanations.